kinto/api-test/mockclient.py

```diff
@@ -2,7 +2,7 @@
 import base64
 import requests
 
-stage_url = "https://kinto.stage.mozaws.net/v1/"
+stage_url = "https://kinto.dev.mozaws.net/v1/"
```

Comment:
some folks refer to both dev and stage as either "dev" or "stage", because they only operate in one environment.  since we operate in 3 (dev, stage, prod), we should always be explicit.  
Perhaps something like:
url_dev, 
url_stage, 
url_prod